keadm/cmd/keadm/app/cmd/common/constant.go

```diff
@@ -79,4 +79,165 @@ const (
+	DescCPU     = "Check node CPU requirements"
+	DescMemory  = "Check node memory requirements"
+	Descdisk    = "Check node disk requirements"
+	DescDNS     = "Check whether DNS can work"
+	DescRuntime = "Check whether runtime can work"
+	DescNetwork = "Check whether the network is normal"
+	DescPID     = "Check node PID requirements"
+
+	CmdGetArch = "arch"
+	//CmdGetCPUNum     = "cat /proc/cpuinfo  |grep processor| wc -l"
+	//CmdGetMenorySize = "free -h |grep Mem|awk '{print $2}'"
+	//CmdGetDiskSize      = "df -h |grep -ve \" /.\"|sed -n \"2p\"|awk '{print $2}'"
+	CmdGetDNSIP         = "cat /etc/resolv.conf |grep nameserver|grep -v ':'|awk '{print $2}'|sed -n '1p'"
+	CmdGetStatusDocker  = "systemctl status docker |grep Active | awk '{print $2}'"
+	CmdPing             = "ping %s -w %d |grep 'packets transmitted' |awk '{print $4}'"
+	CmdGetMaxProcessNum = "sysctl kernel.pid_max|awk '{print $3}'"
+	CmdGetProcessNum    = "ps -A|wc -l"
+
+	AllowedValueCPU     = 1
+	AllowedValueMemory  = 256 * MB
+	AllowedValueDisk    = GB
+	AllowedValuePIDRate = 0.05
+
+	AllowedCurrentValueCPURate  = 0.9
+	AllowedCurrentValueMemRate  = 0.9
+	AllowedCurrentValueDiskRate = 0.9
+
+	AllowedCurrentValueMem  = 128 * MB
+	AllowedCurrentValueDisk = 512 * MB
+
+	KB = 1024
+	MB = KB * 1024
+	GB = MB * 1024
+
+	DefaultDomain = "www.github.com"
+
+	EdgecoreConfig = "config"
+
+	// Default edgecore config path
+	EdgecoreConfigPath = "/etc/kubeedge/config/edgecore.yaml"
+
+	// cmd to copy file
+	CmdCopyFile = "cp -r %s %s/"
+
+	/****Collect****/
+	/*system info*/
+	CmdDiskInfo    = "df -h > %s/disk"
+	CmdArchInfo    = "arch > %s/arch"
+	CmdProcessInfo = "ps -axu > %s/process"
+	CmdDateInfo    = "date > %s/date"
+	CmdUptimeInfo  = "uptime > %s/uptime"
+	CmdHistorynfo  = "history -a && cat ~/.bash_history  > %s/history"
+	CmdNetworkInfo = "netstat -pan > %s/network"
+
+	PathCpuinfo   = "/proc/cpuinfo"
+	PathMemory    = "/proc/meminfo"
+	PathHosts     = "/etc/hosts"
+	PathDNSResolv = "/etc/resolv.conf"
+
+	/*edgecore info*/
+	PathEdgecoreService = "/lib/systemd/system/edgecore.service"
+	CmdEdgecoreVersion  = "edgecore  --version > %s/version"
+
+	/*runtime info*/
+	CmdDockerVersion    = "docker version > %s/version"
+	CmdContainerInfo    = "docker ps -a > %s/containerInfo"
+	CmdContainerLogInfo = "journalctl -u docker  > %s/log"
+	CmdDockerInfo       = "docker info > %s/info"
+	CmdDockerImageInfo  = "docker images > %s/images"
+	PathDockerService   = "/lib/systemd/system/docker.service"
+	/*****************/
+
+	/****Diagnose****/
+	ArgDiagnoseAll  = "all"
+	DescDiagnoseAll = "Diagnose all iteams"
+
+	ArgDiagnoseNode  = "node"
+	DescDiagnoseNode = "Diagnose edge node"
+
+	ArgDiagnosePod  = "pod"
+	DescDiagnosePod = "Diagnose pod"
+
+	ArgDiagnoseInstall  = "install"
+	DescDiagnoseInstall = "Diagnose install"
+	/*****************/
+
+	DefaultRuntime = "docker"
+)
+
+var (
+	AllowedValueArch = []string{"amd64", "arm64v8", "arm32v7", "i386", "s390x", "x86_64", "aarch64"}
```

Comment:
We can't list all the Archs here, I'd suggest remove this check, to avoid misleading users. @shenkonghui @GsssC

Reply:
For example, the arch of my raspberrypi is `armv7l`.

Reply:
Agree : )

Reply:
> We can't list all the Archs here, I'd suggest remove this check, to avoid misleading users. @shenkonghui @GsssC

ok, I wll remove this check next commit

Reply:
Looks good.